scrapy/downloadermiddlewares/ajaxcrawl.py

```diff
@@ -89,8 +88,7 @@ def _has_ajaxcrawlable_meta(text):
     if 'content' not in text:
         return False
 
-    text = _script_re.sub(u'', text)
-    text = _noscript_re.sub(u'', text)
-    text = html.remove_comments(html.replace_entities(text))
+    text = html.remove_tags_with_content(text, ('script', 'noscript'))
```

Comment:
Yeah, you need a website with `<meta name="fragment">` to see the difference, there are shortcuts for websites without this tag, e.g. a website created with wix.com.